symfony/framework-bundle/3.3/config/packages/framework.yaml

```diff
@@ -13,3 +13,13 @@ framework:
     #fragments: ~
     php_errors:
         log: true
+
+    cache:
+        # The app cache caches to the filesystem by default. Other options include:
+
+        # apcu
+        # app: cache.adapter.apcu
```

Comment:
actually, apcu is not really suited to app caches, because its memory model works best with append only use cases (ie perfect fit for cache.system, already used by default)